src/components/Sidebar.vue

```diff
@@ -2,44 +2,117 @@
 import Search from './SidebarComponents/Search.vue'
 import Selection from './SidebarComponents/Selection.vue'
 import Options from './SidebarComponents/Options.vue'
-import { ref } from 'vue'
+import { onMounted, type Ref, ref } from 'vue'
 import { version } from '../../package.json'
 
 const emit = defineEmits<{
 	(event: 'canRenderChart'): void
 }>()
 
+const siteInfo = ref() as Ref<HTMLDialogElement>
 const showOptions = ref(false)
 
 function onCanRender() {
 	emit('canRenderChart')
 	showOptions.value = true
 }
+
+onMounted(async () => {
+	if (typeof HTMLDialogElement !== 'function') {
+		const { default: dialogPolyfill } = await import('dialog-polyfill')
+		dialogPolyfill.registerDialog(siteInfo.value)
+	}
+})
 </script>
 
 <template>
-	<div class="flex-col">
-		<!-- This will be removed post beta -->
-		<div class="mt-1">Site Beta - {{ version }}</div>
-
-		<div class="flex mt-1 mb-1 gap-1 justify-center">
-			<a
-				class="tw-button py-1"
-				href="https://discord.gg/526et4zxBT"
-				target="_blank"
-			>
-				Discord
-			</a>
-			<a
-				class="tw-button py-1"
-				href="https://github.com/buffet-time/MusicChartMaker.com"
-				target="_blank"
+	<div class="flex-col tw-sidebar-width fixed bg-[#303030] h-full">
+		<div>
+			<button
+				type="button"
+				class="tw-button mt-2 mb-1 py-1 px-3"
+				@click="siteInfo.showModal()"
 			>
-				GitHub
-			</a>
+				Site Info
+			</button>
 		</div>
 		<Selection @can-render-chart="onCanRender" />
 		<Options v-if="showOptions" />
 		<Search />
+
+		<dialog ref="siteInfo" class="bg-transparent">
+			<div
+				class="bg-neutral-700 p-5 flex flex-col gap-2 justify-center items-center text-white"
+			>
+				<!-- Header -->
+				<p class="text-neutral-200">Site Beta-{{ version }}</p>
+
+				<p>This site will NEVER have Ads or any form of tracking</p>
+				<p>
+					Disocrd is the preferred platform for discussion and bug reports of
```

Comment:
woops!

Reply:
Lol developer minded people of course can use GitHub but I'd prefer non technical people from posting here :P